homework.txt

```diff
@@ -0,0 +1,41 @@
+Create a calculator 
+
+```
+var num1;
+var num2;
+function calculator (num1, num2, operation) {
+if (operation === 'add' || operation === 'subtract' ||
+ operation === 'multiply' || operation=== 'divide'){
+  var result='';
+  if (operator === 'add') {
+    result = num1 + num2
+  } else if (operator === 'subtract') {
+    result = num1 - num2
+  } else if (operator === 'multiply') {
+    result = num1 * num2
+  } else if (operator === 'divide') {
+    result = num1 / num2
+  }
+  return result;
+   }
+}
+
+```
+
+Write a function to reverse the letters in a word :
+
+```
+var word;
+function reverseString(str) {
+ word = str.split("");
+ word.reverse();
+ str = word.join('');
+ return str;
+}
```

Comment:
Nice, we can also use a loop to get the same result.

Reply:
I have try it ! 

```
function reverseString(str){
    let reverse = "";    
    for (var i = str.length - 1; i >= 0; i--){        
      reverse += str[i];
    }    
    return reverse;
  }
```